src/linux/process.rs

```diff
@@ -38,10 +40,11 @@ pub struct Process {
 }
 
 impl Process {
-    pub fn new(pid: i64, start_time: u64) -> Process {
+    pub fn new(pid: i64, parent: i64, start_time: u64) -> Process {
```

Comment:
A process doesn't always have a parent. `Option<i64>` would be more suitable here.